pyproject.toml

```diff
@@ -81,3 +83,23 @@ find = {}  # Scanning implicit namespaces is active by default
 
 [tool.setuptools.dynamic]
 version = {attr = "pyperformance.__version__"}
+
+[tool.mypy]
+python_version = "3.7"
+pretty = true
+enable_error_code = "ignore-without-code"
+disallow_any_generics = true
+warn_redundant_casts = true
+warn_unused_ignores = true
+warn_unused_configs = true
+files = [
+    'pyperformance/',
+]
+exclude = [
+    'pyperformance/data-files/',
+    'pyperformance/tests/'
+]
+
+[[tool.mypy.overrides]]
+module = "pyperf"
+ignore_missing_imports = true
```

Comment:
pyperf doesn't have type annotations, so without this section, mypy would emit an error at every location pyperf was imported, which was quite annoying.